src/LinkProcessor/LinkParamsProcessor.php

```diff
@@ -83,6 +86,16 @@ public function process(string $class, string $method, array $params): array
             }
         }
 
+        // remove persistent parameters
```

Comment:
We should remove them only if they are not part of method params, I don't see the check here.

Let's say we have persistent param $p, and some actionDefault($a, $b, $p) <- here we should keep $p in url params

Reply:
I think it is processed in previous block if they are part of method params and mobed to $methodParams

Reply:
Yes, but here you will remove them

Reply:
From transferedparams not from methodparams

Reply:
yes, and then they are not returned to LinkProcessor